### PR TITLE
Unit testing

### PR DESCRIPTION
**Pull request by dalloriam**
This PR adds some much-needed unit tests. The pictureflow core & the flow nodes are 100% covered, giving us ~76% test coverage across the whole codebase. We'll still have to test the remaining transform & vision nodes.

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from pictureflow.flow import Broadcast, Combine
+from pictureflow.output import DiskOutput
+from pictureflow.transform import Rotate, Scale
+
+import cv2
+import pictureflow as pf
+
+
+inpt = pf.Placeholder(out_type=pf.Image)
```
Comment (EFF) [inline test.py]: This seems like it's a play around file more than an actual test file do we need it ?

File: test/context.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import os
+import sys
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```
Comment (EFF) [inline test/context.py]: can you add a little comment that explain the reason why you create a test context file

**Review by EFF — APPROVED** [review]
All seems good for unit testing. I'd like to have some higher order test that integrates more than one component / node.

Full usecase example as test would be cool.

File: test.py (hunk 1 above)
Comment (Dalloriam) [reply test.py]: @EFF I just created an [issue](https://app.zenhub.com/workspace/o/mentum/pictureflow/issues/8) for this, I'll get on that soon.

**Pull request merged**